src/test/java/io/managed/services/test/cluster/DataPlaneClusterTest.java

```diff
@@ -0,0 +1,359 @@
+                        log.info("creating an instance");
+                        var payload = new KafkaRequestPayload()
+                            .name(INSTANCE_NAME_PREFIX + placeholderInstanceSuffixCounter.getAndIncrement() + "-" + Environment.LAUNCH_KEY)
+                            .plan(PLAN_STANDARD)
+                            .cloudProvider(Environment.CLOUD_PROVIDER)
+                            .region(Environment.DEFAULT_KAFKA_REGION);
+                        try {
+                            KafkaMgmtApiUtils.attemptCreatingKafkaInstance(kafkaMgmtApi, payload, Duration.ofSeconds(20), Duration.ofSeconds(20));
+                        } catch (KafkaClusterCapacityExhaustedException e) {
+                            log.warn("capacity exhausted at the moment %s", e);
+                        }
+                    }
+
+                    // observe if newly observed value in nodes increased from original snapshot
+                    Map<String, Integer> machineSetToReadyNodeCountSnapshotCurrent = FleetshardUtils.getReadyNodesPerEachMachineSetContainingName(oc, "kafka-standard");
+                    for (var currentlyObservedMachineSet  : machineSetToReadyNodeCountSnapshotCurrent.entrySet()) {
+                        // naming
+                        String machineSetName = currentlyObservedMachineSet.getKey();
+                        int newlyObservedValue = currentlyObservedMachineSet.getValue();
+                        int oldObservedValue = machineSetToReadyNodeCountSnapshotBefore.get(currentlyObservedMachineSet.getKey());
+
+                        // observing if value changed
+                        log.debug("machineSet '{}' capacity before '{}' and now '{}'", machineSetName, oldObservedValue, newlyObservedValue);
+                        if (newlyObservedValue > oldObservedValue) {
+                            log.info("MachineSet '{}' changed scaled number of its nodes", machineSetName);
+                            return true;
+                        }
+                    }
+
+                    // observe if any of newly crated kafka instance really is ready state (node for sure scaled), otherwise continue waiting
+                    return kafkaMgmtApi
+                        .getKafkas(null, null, null, null)
+                        .getItems().stream()
+                        .filter(e -> e.getName().contains(INSTANCE_NAME_PREFIX))
+                        .filter(e -> e.getOwner().equals(Environment.PRIMARY_USERNAME))
+                        .anyMatch(e -> e.getStatus().equals("ready"));
+                }
+            );
+
+        } catch (Exception ignored) {
+        } finally {
+            // delete and wait for cleaning of all instances spawned
+            KafkaMgmtApiUtils.deleteSearchedKafkaInstancesByOwner(kafkaMgmtApi, INSTANCE_NAME_PREFIX, Environment.PRIMARY_USERNAME);
+        }
+    }
+
+    @Test(dataProvider = "managedKafkaTypes")
```

Comment:
The test seems to be specific to `standard` instances: 

1. https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/blob/04c43345024f9dff671221c74a9422b6f5360ca2/src/test/java/io/managed/services/test/cluster/DataPlaneClusterTest.java#L313 

2. https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/blob/04c43345024f9dff671221c74a9422b6f5360ca2/src/test/java/io/managed/services/test/cluster/DataPlaneClusterTest.java#L317

. Are we sure we want to provide a list of mk instance types for this test?

Reply:
changed to count.